actions/parse-push/src/main.ts

```diff
@@ -21,17 +27,17 @@ type Commit = {
  * @param files contains a list of files
  */
 async function getSlugs(files: readonly string[]): Promise<Set<string>> {
-    const re = /exercises\/([a-z-]*)\//
-    let exs = new Set<string>()
+    const pattern = /exercises\/([a-z-]*)\//
+    const slugs = new Set<string>()
 
-    for (const f of files) {
-        const m = re.exec(f)
-        if (m) {
-            exs.add(m[0])
+    for (const file of files) {
+        const matches = pattern.exec(file)
+        if (matches) {
+            slugs.add(matches[0])
         }
     }
 
-    return exs
+    return slugs
```

Comment:
There is nothing pointless about this. `m` and `re` don't mean anything.

Reply:
They mean just as match as `matches` or `re`. It's a lot easier for me to understand and read code with short variable names assuming they're defined close enough. `m` and `re` I can immediately identify at a glance and I can keep track of what they mean in my head. Longer names require actually reading them which makes understanding at a glance much harder for me.

Reply:
The idiomatic way in JavaScript and in TypeScript is to write variable names out, unlike some other languages where it's common (ánd idiomatic) to have short defaults. Because the defaults `re` and `m` don't exist in TS, it's not a good idea _here_. I definitely agree that if it's normal in the language (if `m` and `re` consistently mean something), then one _should_ use short variable names.